src/string.c

```diff
@@ -176,7 +176,7 @@ mrb_str_new_capa(mrb_state *mrb, size_t capa)
 }
 
 #ifndef MRB_STR_BUF_MIN_SIZE
-# define MRB_STR_BUF_MIN_SIZE 128
+# define MRB_STR_BUF_MIN_SIZE 63
```

Comment:
Since the value is already set in `mrbconf.h` lines above are meaningless (my fault).

Reply:
It is set in `mrbconf.h` if` MRB_CONSTRAINED_BASELINE_PROFILE` is defined. This is defined otherwise.
Therefore, it is better to change this when defining it to 31 bytes.